train.py

```diff
@@ -540,16 +548,15 @@ def keep_training(lr, epoch_itr, trainer):
             xm.master_print('old learning rate: {}'.format(lr))
             lr = trainer.lr_step(epoch_itr.epoch, vloss)
             xm.master_print('new learning rate: {}'.format(lr))
+            if args.metrics_debug:
```

Comment:
There is a metrics print ten lines above ...

Reply:
yes, that one is after training is complete. This one is after validation. This helped us catch a few things before, which only occured in validation.

Reply:
Also, because validation is significantly easier/faster than training, it distorts the metrics a little bit, so getting a report right after training is less noisy imho.